client/src/Components/App/App.js

```diff
@@ -1,7 +1,7 @@
 import React, {useState, useEffect} from 'react';
-import { Route, NavLink, Switch } from 'react-router-dom';
+import { Route, Switch } from 'react-router-dom';
+import { getAllEntries } from '../../utilities/api-calls';
 import activityData from '../../sampleData/activityData';
-import entryData from '../../sampleData/entryData';
```

Comment:
[this is really exciting!!!!!!](https://media.giphy.com/media/IwAZ6dvvvaTtdI8SD5/giphy.gif)